
int64 id                               # ID
uint32 type                            # Type
tuw_object_map_msgs/GeoPoint wgs84     # global location in WGS84
geometry_msgs/Point map                # map location in cartesian space
tuw_std_msgs/ParameterFloat64[] parameters                     # parameters to the point


int64 ID_NA = -1
uint32 TYPE_NA = 0